mu/__init__.py

```diff
@@ -8,7 +8,7 @@
 localedir = os.path.abspath(os.path.join(os.path.dirname(__file__), 'locale'))
 language_code = QLocale.system().name()
 # DEBUG/TRANSLATE: override the language code here (e.g. to Chinese).
-# language_code = 'zh'
+# language_code = 'pl'
```

Comment:
To reduce the diff it would be good to revert this change.

Reply:
I'am trying, I'm not a git master ;-)

Reply:
I have made revert - https://github.com/mu-editor/mu/commit/2eaba0f5bed63a0bb7ab0528c593bb337ac81b12
But I do not know what to do next. When I do pull request I see: 
![image](https://user-images.githubusercontent.com/5088643/57582530-22efd380-74c6-11e9-9ea1-cc9083d78e6d.png)

Reply:
That's good, the final diff is only the new flag stuff, so we can always merge and squash to have compress all the final changes into a single commit.